cmd/keycloakb/keycloak_bridge.go

```diff
@@ -1272,33 +1273,21 @@ func config(ctx context.Context, logger log.Logger) *viper.Viper {
 	v.SetDefault(CfgTimeout, "5s")
 
 	// Storage events in DB (read/write)
-	v.SetDefault(CfgAuditRwDbParams+"-enabled", false)
```

Comment:
Why do we remove all those things ?

Reply:
ConfigureDbDefault already initializes the -enabled parameter with value true...
This one is moved 2 lines below to set the parameter to false